mailbox/common/pom.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This commit do much more than what said in comment.
I think you try to reuse some poorly design code (or some code very specific for a task and thus not reusable).